.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing
```

Comment:
for markdown docs, we should try to keep line length at a max of 70. helps a lot with diffs!